packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -343,9 +343,9 @@ export default class ComponentsTableRoute extends Route {
     </:body>
   </Hds::Table>
   <h4 class="dummy-h4">
-    Static table styled as "packed"
+    Static table with density set to "short"
   </h4>
-  <Hds::Table @model={{this.model.data}} @density="packed">
+  <Hds::Table @model={{this.model.data}} @density="short">
```

Comment:
thought: we seem to be in the habit of taking the enum values from the component and [looping through them in the showcase section](https://github.com/hashicorp/design-system/blob/main/packages/components/tests/dummy/app/templates/components/dropdown.hbs#L1041) – I guess it has the advantage of keeping the two in sync with more ease.

Reply:
Ahh yes, I suppose that does happen sometimes. Good catch! 👍